sdk/keyvault/azsecrets/client.go

```diff
@@ -116,7 +116,7 @@ type SetSecretOptions struct {
 	ContentType *string `json:"contentType,omitempty"`
 
 	// The secret management attributes.
-	SecretAttributes *Attributes `json:"attributes,omitempty"`
+	SecretAttributes *Properties `json:"attributes,omitempty"`
```

Comment:
Exposed type should be `SecretProperties`. If this type is only used for de/serialziation, shouldn't it be `secretAttributes`?